server/state.go

```diff
@@ -176,9 +183,39 @@ func (s *SiteVisitController) Begin() {
 	s.game.connection.Broadcast(NewSetClockMessage(SiteVisitDuration))
 	s.game.SetTimeout(SiteVisitDuration)
 
-	s.game.connection.Broadcast(
-		NewEventMessage("it works!", "I'm going to sleep"),
-	)
+	// Set the initial user event queues up.
+	for user, _ := range s.game.UserSites {
+		s.userEventQueue[user] = append(
+			s.userEventQueue[user],
+			NewRepairSite(),
+		)
+	}
+
+	// Try to give all the users their initial events.
+	for user, _ := range s.game.UserSites {
+		s.GiveNewEvent(user)
+	}
+}
+
+func (s *SiteVisitController) GiveNewEvent(u User) {
+	if len(s.userEventQueue[u]) == 0 {
+		fmt.Printf("No events for %q.", u.Name())
```

Comment:
Should send placeholder events for now. 
In the future, send randomly-generated effectless events in the future.

Reply:
Cloud